addons/l10n_cl_hr_expense/models/hr_expense.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = "hr.expense.sheet"
+
+    @api.model
+    def _get_journal_domain(self):
+        default_company_id = self.default_get(['company_id'])['company_id']
+        if self.env['res.company'].browse(default_company_id).country_id.code == 'CL':
+            return [('type', '=', 'general'), ('company_id', '=', default_company_id)]
```

Comment:
This domain won't be the right one if you change the company in the form view (multi-company settings)

Reply:
Thanks @william-andre, made the suggested changes in 606020f8c531b2c7abfe6c19d3fc97848a5bbd45